ohapi/command_line.py

```diff
@@ -14,6 +14,13 @@
 MAX_FILE_DEFAULT = parse_size('128m')
 
 
+def set_log_level(debug, verbose):
```

Comment:
I know this matches the code as it was written, but I'm wondering if we should update this furthter?

That is, should we default to another logging level (WARNING, ERROR, or CRITICAL) when both are False? Do we want behavior to default to INFO (the current "verbose" behavior), and use `--quiet` for less reporting?

Reply:
Yeah, that sounds good. Right now the public download CLI tool behaves different from all others. Making this more consistent would be desirable. But I think that should be tackled in a different PR. :)